extensions/hashicorp-vault/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2022 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH - Initial API and Implementation
+ *
+ */
+
+val okHttpVersion: String by project
+val failsafeVersion: String by project
+
+plugins {
+    `java-library`
+}
+
+dependencies {
+    api(project(":spi:core-spi"))
+
+    implementation("com.squareup.okhttp3:okhttp:${okHttpVersion}")
+    implementation(project(":common:util"))
+    implementation("dev.failsafe:failsafe:${failsafeVersion}")
```

Comment:
not needed, as `RetryPolicy` should be provided by one of the core modules

Reply:
That doesn't seem to be the case.

Reply:
Just verified - it works. pick either one of these variants:
```java
@Inject private RetryPolicy retrypolicy;
@Inject private RetryPolicy<?> retryPolicy;
@Inject private RetryPolicy<Object> retryPolicy;
```
and add it as a class member to the extension.

Reply:
mmh, you're right. that needs to be fixed, since the `CoreServicesExtension` provides the `RetryPolicy`.